tests/e2e/cypress/integration/docs.spec.js

```diff
@@ -250,7 +250,7 @@ describe('Document List', function() {
     cy.get('.col > .col > .col > .Autocomplete > input').click()
     cy.get(
       '.col > .col > .Autocomplete > .Autocomplete-results > .Autocomplete-result'
-    ).click()
```

Comment:
Wouldn’t it be better to use the nth-child selector like you did in the next file?